pkg/api/router/router.go

```diff
@@ -224,6 +224,7 @@ func Router(services *services.Services) (http.Handler, error) {
 	mux.HandleFunc("POST /api/tool-references", toolRefs.Create)
 	mux.HandleFunc("DELETE /api/tool-references/{id}", toolRefs.Delete)
 	mux.HandleFunc("PUT /api/tool-references/{id}", toolRefs.Update)
+	mux.HandleFunc("POST /api/tool-references/{id}", toolRefs.ForceRefresh)
```

Comment:
nit: wondering if it makes it more explicit for this to have `force-refresh` path suffix.

Reply:
Yeah, I'll do that to make it consistent with our other APIs.